progressive-cactus/0.1/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM biocontainers/biocontainers:08252016-4
+
+LABEL base.image="biocontainers:08252016-4"
+LABEL version="0.1"
+LABEL software="progressiveCactus"
+LABEL software.version="08092017"
+LABEL description="multiple whole-genome alignment package"
+LABEL website="https://github.com/glennhickey/progressiveCactus"
+LABEL documentation="https://github.com/glennhickey/progressiveCactus"
+LABEL license="https://github.com/glennhickey/progressiveCactus/blob/master/LICENSE"
+LABEL tags="genomics"
+
+MAINTAINER Ricky Lim <rlim.email@gmail.com>
+
+WORKDIR /home/biodocker/bin
+
+# Download and checkout the source code
+RUN git clone git://github.com/glennhickey/progressiveCactus.git && \
+    cd progressiveCactus && \
+    git checkout tags/0.1 -b 0.1
+
+WORKDIR /home/biodocker/bin/progressiveCactus
+
+RUN git submodule update --init
+
+# Build the source code
+USER root
+
+RUN ln -s /usr/lib/python2.7/plat-*/_sysconfigdata_nd.py /usr/lib/python2.7/ && \
```

Comment:
why do you add this symlink to base python path?

Reply:
aha, this is a little hack around because of the import error
File "/usr/lib/python2.7/_sysconfigdata.py", line 6, in 
from _sysconfigdata_nd import *
ImportError: No module named _sysconfigdata_nd

Reply:
have you check that the hack do not prevent good behavior of the tool? looks like a bug in upstream code.

Reply:
I run the following tests and it runs ok
```
docker image build -t progressive-cactus:0.1 .

# test for help message 
docker run -it --rm progressive-cactus:0.1 

# test for a simulated alignment
docker run -it --rm -w /home/biodocker/bin/progressiveCactus/ -v /tmp/test:/test progressive-cactus:0.1 runProgressiveCactus.sh examples/blanchette00.txt /test /test/b00.hal

# test for a conversion from hal to maf format
docker run -it --rm -w /home/biodocker/bin/progressiveCactus/ -v /tmp/test:/test progressive-cactus:0.1 /bin/bash -c "source environment && hal2mafMP.py /test/b00.hal /test/b00.maf"
```